apps/src/netsim/NetSimMessage.js

```diff
@@ -59,7 +62,8 @@ var NetSimMessage = module.exports = function (shard, messageRow) {
    * All other message content, including the 'packets' students will send.
    * @type {*}
    */
-  this.payload = messageRow.payload;
+  var b64 = messageRow.base64Payload;
+  this.payload = (b64) ? base64ToBinary(b64.string, b64.len) : undefined;
```

Comment:
I'd argue that the parens, although definitely unnecessary, make it a bit clearer that a ternary is being used